docs/docusaurus.config.js

```diff
@@ -1,184 +1,190 @@
-        respectPrefersColorScheme: false,
-      },
-      navbar: {
-        logo: {
-          alt: 'Kurtosis',
-          src: 'img/brand/kurtosis-logo-white-text.png',
-          href: 'https://kurtosis.com',
-          target: '_self'
-        },
-        items: [
-          {
-            to: '/quickstart',
-            position: 'left',
-            label: 'Quickstart',
-            activeBasePath: '/quickstart'
-          },
-          {
-            to: '/cli',
-            position: 'left',
-            label: 'CLI',
-            activeBasePath: '/cli'
-          },
-          {
-            to: '/starlark-reference',
-            position: 'left',
-            label: 'Starlark',
-            activeBasePath: '/sdk'
-          },
-          {
-            href: 'https://web3.kurtosis.com',
-            position: 'left',
-            label: 'Kurtosis for Web3',
-          },
-          {
-            href: 'https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=leeederek&labels=docs&template=docs-issue.yml',
-            position: 'right',
-            label: 'Report Docs Issue',
-          },
-        ],
-      },
-      footer: {
-        style: 'dark',
-        links: [
-          {
-            title: 'Docs',
-            items: [
-              {
-                label: 'Quickstart',
-                to: '/quickstart',
-              },
-              {
-                label: 'CLI',
-                to: '/cli',
-              },
-              {
-                label: 'Starlark',
-                to: '/starlark-reference',
-              },
-              {
-                label: 'Kurtosis for Web3',
-                href: 'https://web3.kurtosis.com',
-              },
-            ],
-          },
-          {
-            title: 'Community',
-            items: [
-              {
-                label: 'Discord',
-                href: 'https://discord.gg/HUapYX9RvV',
-              },
-              {
-                label: 'Twitter',
-                href: 'https://twitter.com/KurtosisTech',
-              },
-              {
-                label: 'GitHub',
-                href: 'https://github.com/kurtosis-tech',
-              },
-            ],
-          },
-          {
-            title: 'Company',
-            items: [
-              {
-                label: `Careers - We're Hiring`,
-                href: 'https://www.kurtosis.com/careers',
-              },
-              {
-                label: 'About Us',
-                href: 'https://www.kurtosis.com/company',
-              },
-              {
-                label: 'Blog',
-                href: 'https://www.kurtosis.com/blog',
-              },
-            ],
-          },
-        ],
-        copyright: `Copyright © ${new Date().getFullYear()} Kurtosis Technologies`,
-      },
-      prism: {
-        theme: lightCodeTheme,
-        darkTheme: darkCodeTheme,
-      },
-      algolia: {
-        appId: 'NTSX40VZB8',
+        ({
```

Comment:
did we go from tabs to spaces?

Reply:
oh i'm just using the auto formatter in intellij. Is tabs our coding standard?

Reply:
I'll fall back to that